pkg/controller/logstorage/elastic/elastic_controller.go

```diff
@@ -0,0 +1,734 @@
+// Copyright (c) 2022-2023 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package elastic
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"time"
+
+	cmnv1 "github.com/elastic/cloud-on-k8s/v2/pkg/apis/common/v1"
```

Comment:
Imports should be reformatted.

Reply:
How so? goimports is telling me this is correct.

Reply:
If you remove the empty lines and sort, and then place empty lines back in the right places, you'll see what I mean.

Reply:
I can do that, but I prefer to keep the API packages all together. I'm separating by function rather than strictly which repository they come from.

Reply:
@tmjd I must be out of the loop then, because this is the way I have tried to organize imports for a long long time. What is the organization scheme you are referring to because it's never been communicated to me!

Reply:
I thought the organization was 3 groups in the following order:
1. language built-in packages
2. 3rd party packages
3. tigera/calico packages
I thought we separated each of those section with a new line. Currently this PR has 5 groups of packages.

Reply:
That's certainly not consistently implemented, or even the majority implemented throughout the codebase. I started going through files to clean them up in my other PR but it was overwhelming the number of files that were not correct, at least according to Rene's suggestion above (which I don't think is the same as what you are saying?)

> If you remove the empty lines and sort, and then place empty lines back in the right places, you'll see what I mean.

All of this comes down to my current and continuing believe that we have no standard here other than what goimports and "make fix" are happy with, and it's probably not reasonable to suddenly start requiring an arbitrary standard before we agree what it is and ideally go through and make existing code match.

Reply:
I have been requesting the standard I've stated when I see people adjusting or reconfiguring the includes in the operator. So I wouldn't say I'm suddenly starting to require an arbitrary standard, it has been the one I've always expected in operator.
This is the format that I think Calico has been using which was carried over from when Calico was implemented in Python.

Reply:
Sure, but one person requesting a format does not a project standard make.

I'm happy to have a discussion on what the standard should be - we have never had that discussion as far as I am aware - and until we  do I just don't think it's fair to say that we have a project standard.